src/ejabberd_sm_mnesia.erl

```diff
@@ -103,7 +103,12 @@ cleanup(Node) ->
                          ['$_']}]),
                 lists:foreach(fun(#session{ usr = {U, S, R}, sid = SID }) ->
                                       mnesia:delete({session, SID}),
-                                      ejabberd_hooks:run(session_cleanup, S, [U, S, R, SID])
+                                      ejabberd_hooks:run_fold(session_cleanup, S,
+                                                              mongoose_acc:new(
```

Comment:
The code could be a bit more readable here if we pass the `Acc` as a variable and construct it just belowe `mnesia:delete`.